[2024-07-03T22:05:01.588+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-03T22:05:02.643+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [queued]>
[2024-07-03T22:05:02.659+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [queued]>
[2024-07-03T22:05:02.709+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-03T22:05:03.039+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_script> on 2024-02-27 00:00:00+00:00
[2024-07-03T22:05:03.242+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_dag', 'run_script', 'scheduled__2024-02-27T00:00:00+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpgbhq6_o8']
[2024-07-03T22:05:03.245+0000] {standard_task_runner.py:91} INFO - Job 102: Subtask run_script
[2024-07-03T22:05:03.319+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=137) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-03T22:05:03.322+0000] {standard_task_runner.py:63} INFO - Started process 146 to run task
[2024-07-03T22:05:04.532+0000] {task_command.py:426} INFO - Running <TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [running]> on host bab0a72648e8
[2024-07-03T22:05:19.697+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juan_ml' AIRFLOW_CTX_DAG_ID='exchange_rate_dag' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2024-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-27T00:00:00+00:00'
[2024-07-03T22:05:19.706+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-03T22:05:19.711+0000] {logging_mixin.py:188} INFO - All credentials are set properly.
[2024-07-03T22:05:19.715+0000] {logging_mixin.py:188} INFO - Error al crear la tabla en Redshift: invalid literal for int() with base 10: 'None'
[2024-07-03T22:05:20.862+0000] {logging_mixin.py:188} INFO - Error al realizar la solicitud a la API: 401 Client Error: Unauthorized for url: https://api.apilayer.com/exchangerates_data/latest?symbols=EUR,USD,CNY&base=ARS
[2024-07-03T22:05:20.868+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-03T22:05:20.871+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-03T22:05:20.959+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=exchange_rate_dag, task_id=run_script, run_id=scheduled__2024-02-27T00:00:00+00:00, execution_date=20240227T000000, start_date=20240703T220502, end_date=20240703T220520
[2024-07-03T22:05:21.423+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-03T22:05:21.509+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T22:05:21.522+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-03T22:50:50.036+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-03T22:50:50.117+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [queued]>
[2024-07-03T22:50:50.153+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [queued]>
[2024-07-03T22:50:50.155+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-03T22:50:50.229+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_script> on 2024-02-27 00:00:00+00:00
[2024-07-03T22:50:50.263+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=167) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-03T22:50:50.272+0000] {standard_task_runner.py:63} INFO - Started process 181 to run task
[2024-07-03T22:50:50.270+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_dag', 'run_script', 'scheduled__2024-02-27T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpj29927x1']
[2024-07-03T22:50:50.276+0000] {standard_task_runner.py:91} INFO - Job 29: Subtask run_script
[2024-07-03T22:50:50.429+0000] {task_command.py:426} INFO - Running <TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [running]> on host 0b7358cba041
[2024-07-03T22:50:51.333+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juan_ml' AIRFLOW_CTX_DAG_ID='exchange_rate_dag' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2024-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-27T00:00:00+00:00'
[2024-07-03T22:50:51.339+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-03T22:50:51.347+0000] {logging_mixin.py:188} INFO - All credentials are set properly.
[2024-07-03T22:50:51.349+0000] {logging_mixin.py:188} INFO - Error al crear la tabla en Redshift: invalid literal for int() with base 10: 'None'
[2024-07-03T22:50:52.510+0000] {logging_mixin.py:188} INFO - Error al realizar la solicitud a la API: 401 Client Error: Unauthorized for url: https://api.apilayer.com/exchangerates_data/latest?symbols=EUR,USD,CNY&base=ARS
[2024-07-03T22:50:52.514+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-03T22:50:52.516+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-03T22:50:52.539+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=exchange_rate_dag, task_id=run_script, run_id=scheduled__2024-02-27T00:00:00+00:00, execution_date=20240227T000000, start_date=20240703T225050, end_date=20240703T225052
[2024-07-03T22:50:52.615+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-03T22:50:52.656+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-03T22:50:52.661+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-04T01:38:44.855+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-04T01:38:44.891+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [queued]>
[2024-07-04T01:38:44.905+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [queued]>
[2024-07-04T01:38:44.906+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-04T01:38:44.927+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_script> on 2024-02-27 00:00:00+00:00
[2024-07-04T01:38:44.942+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=385) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-04T01:38:44.944+0000] {standard_task_runner.py:63} INFO - Started process 387 to run task
[2024-07-04T01:38:44.945+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_dag', 'run_script', 'scheduled__2024-02-27T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmph8qewfi_']
[2024-07-04T01:38:44.947+0000] {standard_task_runner.py:91} INFO - Job 28: Subtask run_script
[2024-07-04T01:38:45.021+0000] {task_command.py:426} INFO - Running <TaskInstance: exchange_rate_dag.run_script scheduled__2024-02-27T00:00:00+00:00 [running]> on host e26a552f405a
[2024-07-04T01:38:45.169+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juan_ml' AIRFLOW_CTX_DAG_ID='exchange_rate_dag' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2024-02-27T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-27T00:00:00+00:00'
[2024-07-04T01:38:45.171+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-04T01:38:45.173+0000] {logging_mixin.py:188} INFO - All credentials are set properly.
[2024-07-04T01:38:45.173+0000] {logging_mixin.py:188} INFO - Error al crear la tabla en Redshift: invalid literal for int() with base 10: 'None'
[2024-07-04T01:38:45.629+0000] {logging_mixin.py:188} INFO - Error al realizar la solicitud a la API: 401 Client Error: Unauthorized for url: https://api.apilayer.com/exchangerates_data/latest?symbols=EUR,USD,CNY&base=ARS
[2024-07-04T01:38:45.635+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-04T01:38:45.637+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-04T01:38:45.670+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=exchange_rate_dag, task_id=run_script, run_id=scheduled__2024-02-27T00:00:00+00:00, execution_date=20240227T000000, start_date=20240704T013844, end_date=20240704T013845
[2024-07-04T01:38:45.845+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-04T01:38:45.958+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-04T01:38:45.970+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
